protocols/gossipsub/src/behaviour.rs

```diff
@@ -1454,7 +1454,7 @@ impl fmt::Debug for GossipsubRpc {
 }
 
 /// Event that can happen on the gossipsub behaviour.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
I won't add a changelog entry for this. In case you think it is necessary, please speak up.